calico-cloud/reference/resources/bgppeer.mdx

```diff
@@ -57,6 +57,7 @@ spec:
 | numAllowedLocalASNumbers | The number of local AS numbers to allow in the AS path for received routes. This disables BGP loop prevention and should only be used if necessary.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                                             | integer                     | `nil` (BIRD will default to 0 meaning no change to loop prevention behavior) |
 | ttlSecurity              | Enables the generalized TTL security mechanism (GTSM) which protects against spoofed packets by ignoring received packets with a smaller than expected TTL value. The provided value is the number of hops (edges) between the peers.                                                                                                                                                                                                                                                                                                                                                                                                                      | 0 - 255                                                                                     | 8-bit integer               | `nil` (results in BIRD configuration `ttl security off`)                     |
 | filters                  | List of names of [BGPFilter](bgpfilter.mdx) resources to apply to this peering.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |                                                                                             | string                      |                                                                              |
+| reversePeering | Specifies whether Calico automatically generates reverse peerings for nodes selected by the peerSelector. If set to Manual, a separate BGPPeer must be created for the reverse peering. | "Auto", "Manual" | string | "Auto" |
```

Comment:
@song This CC BGPPeer spec appears to be out of date. Should this be in line with the corresponding CE doc?

Reply:
Yes. Good catch. Thanks.